### DISCUSSION TITLE
导入太频繁，问题排查

### DISCUSSION THREAD

**Discussion by yhp353134** [apache/doris#6723] [A - General / Q&A]
dorisWriter 频繁导入，导致be挂了，在服务器找不到任何日志来定位这个问题：
1、dmesg -T | grep doris  无记录
2、be.INFO也没有任何记录，请问这个怎么来定位这个问题

**Comment by morningman** [comment]
dmesg -T 应该grep palo_be，palo_be 这个是进程名
